website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,167 @@
+
+![Search enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-search.png =836x*)
+
+##### Search to add behavior
+If enabled, searched terms can be added to the list. This feature is only applicable to the multiple variant.
+![Showing the search to add behavior in the multiple variant of SuperSelect](/assets/components/form/super-select/super-select-multi-flow-search-to-add.png =836x*)
+
+#### CustomContent
+If enabled, yielded content will occupy this space.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-custom-content.png =836x*)
+
+!!! Dont
+
+We recommend against placing heavy UI elements in the custom content as it can draw attention away from the list options and will only be visible when the menu is open. Consider placing them in relation to the trigger.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-dont-visually-heavy.png =836x*)
+
+!!!
+
+### AfterOptions
+AfterOptions allows for one of two options to be displayed at a given time: footer or custom content. These options are the same between both single and multiple variants, however the footer has additional functions in the multiple variant.
+
+#### Footer
+The footer appends below the list displaying content and/or buttons (depending on the variant) that provide additional context (or functions) to the list options.
+
+##### Single variant
+The footer defaults to visible and shows a total counter, which is customizable to fit context.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png =626x*)
+
+##### Multiple variant
+The afterOptions and footer default to visible. The buttons allow users to expedite the deselection process. The counter (selected / total) is optional. Either both or individually may be shown.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-multiple-variant-footer-without-counter.png =757x*)
+
+Show selected will change the list to display only selected list items while toggling the label of that button to “Show all."
+
+![Flow showing the behavior of the show selected button and how it toggles the list to only display selected list items](/assets/components/form/super-select/super-select-flow-select-all.png =836x*)
+
+If show selected is used while no selections have been made, a string of text will let the user know that no selections have been made yet.
+
+![Flow showing the behavior of the show selected button when no selections have been made yet.](/assets/components/form/super-select/super-select-flow-no-selections-select-all.png =836x*)
```

Comment:
All of these examples have text burned into the images, I might recommend finding a way to abstract this out of the images just because it's easier to update that way. Potentially could use _emphasized_ formatting here, that's somewhat common for other components.

Reply:
Removed text and abstracted. I think the paragraphs prior to the images help explain the flow below.